tests/components/application-connector/k8s/gateway-test/templates/_helpers.tpl

```diff
@@ -0,0 +1,27 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
```

Comment:
name and fullname templates are not used

Reply:
Removed